nerfstudio/engine/schedulers.py

```diff
@@ -25,92 +26,71 @@
 
 @dataclass
 class SchedulerConfig(InstantiateConfig):
-    """Basic scheduler config with self-defined exponential decay schedule"""
+    """Basic scheduler config"""
 
-    _target: Type = field(default_factory=lambda: ExponentialDecaySchedule)
+    _target: Type = field(default_factory=lambda: Scheduler)
+    """_target: target class to instantiate"""
+
+
+class Scheduler:  # pylint: disable=too-few-public-methods
+    """Exponential learning rate decay function."""
```

Comment:
@tancik looks like you missed a docstring?

Reply:
Thanks for catching